modules/Microsoft.DataFactory/factories/managedVirtualNetwork/deploy.bicep

```diff
@@ -29,6 +32,19 @@ resource managedVirtualNetwork 'Microsoft.DataFactory/factories/managedVirtualNe
   properties: {}
 }
 
+module managedVirtualNetwork_managedPrivateEndpoint 'managedPrivateEndpoints/deploy.bicep' = [for (managedPrivateEndpoint, index) in managedPrivateEndpoints: {
+  name: '${deployment().name}-managedPrivateEndpoint-${index}'
+  params: {
+    dataFactoryName: dataFactoryName
+    managedVirtualNetworkName: name
+    name: managedPrivateEndpoint.name
+    fqdns: managedPrivateEndpoint.fqdns
+    groupId: managedPrivateEndpoint.groupId
+    privateLinkResourceId: managedPrivateEndpoint.privateLinkResourceId
+    enableDefaultTelemetry: enableDefaultTelemetry
```

Comment:
This should reference a variable `var enableReferencedModulesTelemetry = false`

Reply:
Fixed